README.MD

```diff
@@ -1,9 +1,39 @@
-# Designing our own Microprocessor
+# Super32 - Designing our own Microprocessor
 
-## References
+This repository contains the source code for a simple assembler / compiler for our educational processor.
 
-* [LLVM Compiler Infrastructure](https://llvm.org/)
-    * [UserGuides](https://llvm.org/docs/UserGuides.html)
-    * [Target-Independent Code Generator](https://llvm.org/docs/CodeGenerator.html)
-    * [TableGen](https://llvm.org/docs/TableGen/index.html)
-* [GNU Binutils](https://sourceware.org/binutils/docs-2.33.1/)
+At the moment this repository consists of two projects. py-assembler is an assembler, written in [Python](https://www.python.org/) and Super32 is an attempt to implement an actuall backend compiler with [LLVM](https://llvm.org/).
+
+## Getting Started
+
+These instructions will help you to get your environment ready to build, develop and run the project on your local machine.
+
+### Prerequisites
+
+You'll need the following tools for py-assembler:
+
+- [Python 3](https://www.python.org/)
+- Python modules defined in requirements.txt
+
+For the Super32 project you need:
+
+- [LLVM](https://llvm.org/)
+
+### Run py-assembler
+
+To use the python assembler you must change directory to py-assembler and run main.py with python. To actually assemble code you have to provide an input file via argument.
+
+```Bash
+cd py-assembler input.txt
```

Comment:
delete input.txt in this line and add it as argument to `python main.py input.txt` in next line.

Reply:
Yea my mistake. I completely overlooked that.